miner/miner.go

```diff
@@ -136,6 +140,10 @@ func (self *Miner) Stop() {
 	self.worker.stop()
 	atomic.StoreInt32(&self.mining, 0)
 	atomic.StoreInt32(&self.shouldStart, 0)
+	mlogMiner.Send(mlogMinerStop.SetDetailValues(
+		self.coinbase.Hex(),
+		self.threads,
+	))
```

Comment:
Still the same. What happens when `mlog=off`?

Reply:
Just for reference this is what happens:

```go
// SendMLog writes enabled component mlogs to file if the component is registered active.
func (c mlogComponent) Send(msg MLogT) {
	mlogRegLock.RLock()
	if l, exists := MLogRegistryActive[c]; exists {
		l.SendFormatted(GetMLogFormat(), 1, msg)
	}
	mlogRegLock.RUnlock()
}

func (l *Logger) SendFormatted(format mlogFormat, level LogLevel, msg MLogT) {
	switch format {
	case mLOGKV:
		l.Sendln(level, msg.FormatKV())
	case MLOGJSON:
		logMessageC <- stdMsg{level, string(msg.FormatJSON())}
	case mLOGPlain:
		l.Sendln(level, string(msg.FormatPlain()))
	//case MLOGDocumentation:
		// don't handle this because this is just for one-off help/usage printing documentation
	default:
		glog.Fatalf("Unknown mlog format: %v", format)
	}
}
```